tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -49,10 +49,14 @@ sub run {
             timeout => 3600,
             verbose => 1);
         my $find_cmd = join(' ',
-            'find',
+            'timeout', '10',
+            'find', '-P', '-O3',
             '/tmp/results/',
             '-type', 'f',
-            '-iname', "*.xml");
+            '-iname', '*.xml',
+            '-print', '2>/dev/null;',
+            '[', '$?', '-ne', '124', ']',
+            '||', 'echo', '"Command find timed out"');
```

Comment:
OK, with the added `timeout 10` and `|| echo "Command find timed out"`, there's a guarantee that `$find_cmd` will end within 10 seconds, but shouldn't we also check around L73 whether we got `Command find timed out` as a result of `script_output()`?